blocklylib-core/src/main/java/com/google/blockly/model/Input.java

```diff
@@ -100,7 +100,7 @@
         INPUT_TYPES.add(TYPE_DUMMY_STRING);
     }
 
-    private final String mName;
+    private final @Nullable String mName;
```

Comment:
Why is the input name nullable? This would create an error when serializing/deserializing and in the generators.